Makefile

```diff
@@ -53,40 +101,22 @@ setup-test-files:
 go-install:
 	go install
```

Comment:
This results in $GO_PATH/bin/kustomize-sops

Wouldn't it be better to combine this with go build that produces the same thing as ksops? Then we only need to symlink one of them, reducing build time, and making the image smaller. Or am I missing some magic here?

@devstein

Reply:
you are right